### ISSUE TITLE
Modify a process instance by activating elements and setting global variables

### ISSUE THREAD

**Issue by saig0** [camunda/zeebe#9663]
## Description

We want to pass variables when activating an element. The variables should be stored globally, in the scope of the process instance. Each activation instruction can contain variables. 

The variables should be set before activating the element. If the element is embedded in a subprocess then the variables should be set before activating the scope. The variables may be required to create event subscriptions of the element or of an upper flow scope.

For example:
![image](https://user-images.githubusercontent.com/4305769/176903468-af96801a-3fd9-41d8-8e8f-2284f117b9fc.png)

* set the variables by writing a `created` command for each activate instruction
  * set the process instance key as the scope key of the variables

Out of scope:
* set local variables 

Blocked by 
- #9643
